core/src/main/java/io/grpc/internal/MutableHandlerRegistry.java

```diff
@@ -29,7 +29,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc;
+package io.grpc.internal;
```

Comment:
So are you ok with the change? It seems that making it internal for now is a reasonable solution given that nothing else in the public API uses it ATM.